register.js

```diff
@@ -133,17 +133,22 @@
 
 	// Returns an array of global event handlers names
 	// https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers
-	function getGlobalEventHandlersNames() {
-		var names = [];
-
-		if (!isNode) {
-			names =  Object.getOwnPropertyNames(HTMLElement.prototype).filter(
-				isGlobalEventHandler
-			);
-		}
-
-		return names;
-	}
+  function getGlobalEventHandlersNames() {
+    var names = [];
+    var isWorkerScope = typeof WorkerGlobalScope !== "undefined" &&
+                        typeof self !== "undefined" &&
+                        self instanceof WorkerGlobalScope;
+
+    if (!isNode && !isWorkerScope) {
+      names = Object.getOwnPropertyNames(HTMLElement.prototype).filter(
+        function isGlobalEventHandler(name) {
```

Comment:
Why inline this function? it's already defined here: https://github.com/canjs/can-zone/blob/0507a64fb92870983a836681a48eaa840b43b38b/register.js#L130

Reply:
Oh, bad copypasta from tasks.js.  I'll fix that.